dpnp/dpnp_iface_manipulation.py

```diff
@@ -583,12 +583,28 @@ def rollaxis(x1, axis, start=0):
     return call_origin(numpy.rollaxis, x1, axis, start)
 
 
-def squeeze(x1, axis=None):
+def squeeze(x, axis=None):
```

Comment:
to align with python array API
```suggestion
def squeeze(x, /, axis=None):
```